README.md

```diff
@@ -180,10 +193,11 @@ the CMS detector in pp collisions at 13 TeV](https://inspirehep.net/record/16443
 
 - T. Cheng, "[Recursive Neural Networks in Quark/Gluon Tagging](https://inspirehep.net/record/1634876)," arXiv:1711.02633 [hep-ph]. (November 7, 2017)
 
-- M. Frate, K. Cranmer, S. Kalia, A. Vandenberg-Rodes, and D. Whiteson, “[Modeling Smooth Backgrounds and Generic Localized Signals with Gaussian Processes](https://inspirehep.net/record/1624168),” arXiv:1709.05681 [physics.data-an]. (September 17, 2017)
+- S. Chang, T. Cohen, and B. Ostdiek, "[What is the Machine Learning?](https://inspirehep.net/record/1627883)," arXiv:1709.10106 [hep-ph]. (September 28, 2017)
```

Comment:
@matthewfeickert All of the changes (other than adding the paper) were made by tidy-markdown. I will make a new version not using that will also incorporate the changes in the headings which you mentioned.

Reply:
@matthewfeickert I think these should be in the correct format now

Reply:
LGTM! :shipit: